geoutils/raster/raster.py

```diff
@@ -28,6 +28,7 @@
 from rasterio.enums import Resampling
 from rasterio.plot import show as rshow
 
+import geoutils as gu
```

Comment:
Does it not lead to a circular import?? Would `from geoutils import Vector` be better?